Asterix/tuto_asterix_model.py

```diff
@@ -152,7 +155,7 @@
 
 # Finally, we can initialize some DMs. DMs can be in a pupil or outside a pupil.
 # In the default configuraiton file, this one is in a pupil plane.
-DM3 = OptSy.DeformableMirror(modelconfig,
+DM3 = DeformableMirror(modelconfig,
```

Comment:
Don't forget to fix the indentation in cases like this.

Reply:
we agreed to do yapf format in a separate PR so that it's easier to read

Reply:
Then I might have misunderstood: I thought we'd implement linting "light", meaning just running to see what's amiss, not actually reformatting code, I agreed with you when we talked about it I would rather only let humans do/trigger that.
What I am saying here is that if it already is nicely formatted (indented in this case), a code change should preserve that. Otherwise you are just going back and forth between enforcing a certain formatting and not enforcing it.

Reply:
not sure to understand what you mean

Reply:
i care about indenting if it might send an error (if / for loops, etc). I don't care in the function parameters like here and I'll ran a full yapf formating for all functions in a separate PR after

Reply:
But... why do a separate PR for formatting if you could just run it now?

Reply:
Remember how `master` is always supposed to have a working version of the code? "working" in this context would also mean in the default format - so as much as broken code should not be merged, mis-formatted code shouldn't be merged. Especially easy if you have an automatic formatter, no? :)